docs/package_templates/header_only/all/conanfile.py

```diff
@@ -57,22 +57,18 @@ def requirements(self):
     def package_id(self):
         self.info.clear()
 
-    @property
-    def _info(self):
-        return self if Version(conan_version).major < 2 else self.info
```

Comment:
I surprised we had it in the first place. I am glad it removed. recipes shouldn't contain logic to detect v1/v2. they should just use a sub-set of v1/v2 features compatible with both.